packages/util/test/blobHelpers.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { assert, describe, it } from 'vitest'
+
+import { commitmentsToVersionedHashes, computeVersionedHash, getBlobs } from '../src/index.js'
+
+describe('YourClass', () => {
```

Comment:
Total nit but can we rename this to something that reflects the test case, just `blob helper tests` at a minimum?

Reply:
Fixed.